packages/fiori-elements-writer/src/data/annotationReuseLibs.ts

```diff
@@ -0,0 +1,58 @@
+import { OdataVersion } from '../types';
+
+interface AnnotationReuseLibsEntry {
+    annotation: string;
+    reuseLib: string;
```

Comment:
Should this be just `library` as we cannot tell if its a reuse lib or not, it just happens that the one being added is a reuse lib. Or will every annotation related library in future be a reuse lib?

Reply:
Thanks, I have added your suggestion.